app/services/joke-service.server.ts

```diff
@@ -108,4 +107,14 @@ export class JokeService extends BaseService {
       data: { queued, queuedAt: queued ? new Date() : null },
     });
   }
+
+  public async deleteJoke(jokeId: number): Promise<void> {
+    const joke = await db.joke.findUnique({ where: { id: jokeId } });
```

Comment:
@Mando75 is there a better way that you know if with prisma to do the submitterId check at the same time? The only thing I can think of is to make an index that includes both submitterId and id.

Reply:
😂 salty